rclcpp/test/test_duration.cpp

```diff
@@ -60,6 +60,7 @@ TEST(TestDuration, operators) {
   rclcpp::Duration time = rclcpp::Duration(0, 0);
   rclcpp::Duration copy_constructor_duration(time);
   rclcpp::Duration assignment_op_duration = rclcpp::Duration(1, 0);
+  (void)assignment_op_duration;
```

Comment:
Because otherwise it is unused, again this is a quirk of cppcheck when it is fed a single file (probably cannot tell if `rclcpp::Duration` is RAII or not).